### PR TITLE
changed the NTP server 

### PR DESCRIPTION
**Pull request by erluxman**
changed the NTP server to time.google.com coz 0.north-america.pool.ntp.org was causing InvalidNtpServerResponseException.

 
--

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
-        <activity android:name=".Sample2Activity">
+        <activity android:name=".SampleActivity">
```
Comment (kaushikgopal) [inline app/src/main/AndroidManifest.xml]: could you switch this back to `Sample2Activity`. That one demonstrates true NTP (vs SNTP)

Comment (erluxman) [inline app/src/main/AndroidManifest.xml]: Sure

**Comment by kaushikgopal** [comment]
🙏  thanks for this

**Comment by erluxman** [comment]
@kaushikgopal  You are welcome but I shall thank you more for the opportunity  :pray:

**Pull request merged**